cogs/user.py

```diff
@@ -283,7 +283,7 @@ async def vc(self, ctx, *usernames):
         )
         await ctx.send(embed=embed)
 
-    def force(argument) -> typing.Optional[bool]:
+    def force(self, argument) -> typing.Optional[bool]:
```

Comment:
```suggestion
    def force(argument) -> typing.Optional[bool]:
```

I intentionally did not add self in the first place